python/cuml/test/explainer/test_gpu_treeshap.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import treelite
+import numpy as np
+from cuml.experimental.explainer.tree_shap import TreeExplainer
+from cuml.common.import_utils import has_xgboost
+from cuml.ensemble import RandomForestRegressor as curfr
+from sklearn.datasets import fetch_california_housing, make_classification
+
+if has_xgboost():
+    import xgboost as xgb
+
+
```

Comment:
Would be nice to test some edge cases for at least one model type.
e.g.
Rf model that has not called fit yet.
Empty model.
Empty input (user asks for shap values for zero rows).